components/search/Searchbar.tsx

```diff
@@ -116,66 +142,55 @@ function Searchbar({
+        }`}
       >
-        <div class="gap-4 grid grid-cols-1 sm:grid-rows-1 sm:grid-cols-[150px_1fr]">
-          <div class="flex flex-col gap-6">
-            <span
-              class="font-medium text-xl"
-              role="heading"
-              aria-level={3}
-            >
-              Sugestões
-            </span>
-            <ul id="search-suggestion" class="flex flex-col gap-6">
-              {searches.map(({ term }) => (
-                <li>
-                  <a href={`/s?q=${term}`} class="flex gap-4 items-center">
-                    <span>
-                      <Icon
-                        id="MagnifyingGlass"
-                        size={24}
-                        strokeWidth={0.01}
-                      />
-                    </span>
-                    <span dangerouslySetInnerHTML={{ __html: term }} />
-                  </a>
-                </li>
-              ))}
-            </ul>
-          </div>
-          <div class="flex flex-col pt-6 md:pt-0 gap-6 overflow-x-hidden">
-            <span
-              class="font-medium text-xl"
-              role="heading"
-              aria-level={3}
-            >
-              Produtos sugeridos
-            </span>
-            <Slider class="carousel">
-              {products.map((product, index) => (
-                <Slider.Item
-                  index={index}
-                  class="carousel-item first:ml-4 last:mr-4 min-w-[200px] max-w-[200px]"
+        <div class="p-0 sm:p-5 pt-5 font-galano lowercase w-full">
+          <ul class="sm:text-sm text-base font-light w-full">
+            {suggestions.value?.searches?.map((search) => (
+              <li class="w-full ">
+                <a
+                  href={`/s?q=${search.term}`}
+                  class="hover:underline leading-[26px] py-[5px] pr-5 cursor-pointer flex flex-row"
                 >
-                  <ProductCard
-                    product={product}
-                    platform={platform}
-                    index={index}
-                    itemListName="Suggeestions"
-                  />
-                </Slider.Item>
-              ))}
-            </Slider>
+                  <span class="font-semibold">
+                    {searchInputRef.current?.value}
+                  </span>
+                  {serachTermRemove(search.term)}
+                </a>
+              </li>
+            ))}
+          </ul>
+          <label class="flex items-center">
+            <div class="flex-1 h-px bg-[#e5e5e5] sm:bg-black"></div>
+            <div class="px-5 text-lg sm:text-2xl">produtos sugeridos</div>
+            <div class="flex-1 h-px bg-[#e5e5e5] sm:bg-black"></div>
+          </label>
+          <div class="grid gap-5 grid-cols-3">
+            {suggestions.value?.products?.map((product, index) => (
+              <ProductCardSuggestion
+                product={product}
+                index={index}
+              />
```

Comment:
https://preactjs.com/tutorial/08-keys/

```suggestion
            {suggestions.value?.products?.map((product, index) => (
              <ProductCardSuggestion
                product={product}
                index={index}
                key={index}
              />
```